network_importer/adapters/netbox_api/models.py

```diff
@@ -33,6 +33,35 @@
 
 LOGGER = logging.getLogger("network-importer")
 
+from diffsync.enum import DiffSyncModelFlags
+
+def _is_tag_present(obj):
+    """Find if tag is present for a given object."""
+    
+    tags = list()
+    for tag in config.SETTINGS.main.model_flag_tags:
+        if tag in obj['tags']:
```

Comment:
Would be good to check if tags is present on the object first and is not null, just in case the format of the object changes in the future

Reply:
Updated